content/docs/dvclive/user-guide/dvclive-with-dvc.md

```diff
@@ -0,0 +1,129 @@
+# DVCLive with DVC
+
+Even though DVCLive does not require DVC, they can integrate in several useful
+ways:
+
+- The [_outputs_](#outputs) DVCLive produces can be fed as
+  `dvc plots`/`dvc metrics`, making it easier to add metrics logging to DVC
+  <abbr>stages</abbr>. Those same outputs can be visualized in
+  [_DVC Studio_](#dvc-studio)
+- You can monitor model performance in realtime with the
+  [_HTML report_](#html-report) that DVCLive generates when used alongside DVC.
+- DVCLive is also capable of generating [_checkpoint_](#checkpoints) signal
+  files used by DVC <abbr>experiments<abbr>.
+
+## Setup
+
+We will refer to a training script (`train.py`) already using `dvclive`:
+
+```python
+# train.py
+
+import dvclive
+
+for epoch in range(NUM_EPOCHS):
+    train_model(...)
+    metrics = evaluate_model(...)
+
+    for metric_name, value in metrics.items():
+        dvclive.log(metric_name, value)
+
+    dvclive.next_step()
+```
+
+> 💡 When using DVCLive in a DVC project, there is no need for manual
+> initialization of DVCLive (no [`dvclive.init()`] call). DVCLive will use the
+> settings provided by DVC.
```

Comment:
Probably not necessary to call this out, and it sounds a bit like `dvclive.init()` is required otherwise.